src/simulator.py

```diff
@@ -711,9 +712,16 @@ def simulation(mode, out, dna_type, per, kmer_bias, max_l, min_l, median_l=None,
     # Start simulation
     sys.stdout.write(strftime("%Y-%m-%d %H:%M:%S") + ": Start simulation of random reads\n")
```

Comment:
Please change the order of log information. At this stage, it's not really simulating random reads yet.